deploy.sh

```diff
@@ -82,41 +53,64 @@ build_push_images() {
 	make push-manifest
 }
 
-if [ "${TRAVIS_BRANCH}" != 'devel' ]; then
-	echo "!!! Branch ${TRAVIS_BRANCH} is not a deployable branch; exiting"
-	exit 0 # Exiting 0 so that this isn't marked as failing
-fi
+push_helm_charts() {
+	PACKAGE=$1
+	CHARTDIR=$2
+	VERSION=${CSI_IMAGE_VERSION//v/} # Set version (without v prefix)
 
-if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
-	build_step "log in to quay.io as user ${QUAY_IO_USERNAME}"
-	# This is a workaround to fix docker permission denied issue during manifest create in Travis CI.
-	# `docker manifest create` fails due to permission denied on `/etc/docker/certs.d/quay.io`
-	# (https://github.com/docker/for-linux/issues/396).
-	sudo chmod o+x /etc/docker
+	# update information in Chart.yaml if the branch is not devel
+	if [ "${BRANCH_NAME}" != "devel" ]; then
+		# Replace appVersion: canary and version: *-canary with the actual version
+		sed -i "s/\(\s.*canary\)/ $VERSION/" "charts/ceph-csi-$PACKAGE/Chart.yaml"
```

Comment:
Still many bash variables without `${..}`

Reply:
these are not added as part of this PR but let me fix it.